src/ui/map.js

```diff
@@ -1168,6 +1172,10 @@ class Map extends Camera {
         }
     }
 
+    _translateUiString(key: string) {
+        return this._locale[key] || key;
```

Comment:
Nit: is there a case where we want `key` to be returned if the input is supposed to be an id rather than a final string? If not, perhaps we should rename to `_getUIString`, and throw an error if `this._locale[key]` is not defined.